### ISSUE TITLE
Support releases deployment

### ISSUE THREAD

**Issue by ChialiT** [Unirep/explorer#24]
We are targeting to release a public version V0.1 on Feb 22, would need to have following support:
- create a URL : "explorer.unirep.io"
- FE will be host on cloudflare, please update DNS as well
- (cluster EC2 - need more info from Chance)

**Comment by vimwitch** [comment]
Yes, I'm waiting to deploy to a more public url until the explorer is a bit more polished (and we have a final testnet deployment for beta-1). Making it public should take a couple minutes max

**Comment by jp4g** [comment]
Is there any work around hosting a back-end synchronizer? This might take a bit longer for me to set up perfectly, being less familiar with Unirep.

Otherwise for just DNS on front end using their auto load-balancing takes 7 seconds to do right. I could potentially set up a ci/cd pipeline for this action?

**Comment by vimwitch** [comment]
Not really, we have it already running, and a ci [here](https://app.circleci.com/pipelines/github/Unirep/explorer/38/workflows/fd7bf74e-0bac-4932-9da2-6d1dabb801a5/jobs/47). We might want a separate deployment for like staging vs what is public in the future.
